simple-wp-helpdesk/frontend/class-portal.php

```
@@ -240,172 +351,24 @@ function ( $u ) use ( $ticket_id ) {
-			$attach_urls  = get_comment_meta( (int) $comment->comment_ID, '_attachments', true );
-			$reply_names  = get_comment_meta( (int) $comment->comment_ID, '_swh_reply_orignames', true );
-			$reply_names  = is_array( $reply_names ) ? $reply_names : array();
-			$gmt_str      = $comment->comment_date_gmt;
-			$utc_ts       = $gmt_str ? strtotime( $gmt_str . ' UTC' ) : false;
-			$utc_ts       = false !== $utc_ts ? $utc_ts : time();
-			$comment_iso  = gmdate( 'c', $utc_ts );
-			$comment_disp = swh_format_comment_date( $comment );
+	// v4.0.0 #354 — Portal v2 messenger layout.
+	$meta_status = get_post_meta( $ticket_id, '_ticket_status', true );
+	$meta_prio   = get_post_meta( $ticket_id, '_ticket_priority', true );
+	$title       = $post->post_title;
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$title'. (undefined)

(UnusedLocalVariable)